### PR TITLE
[fix](regression) fix inverted_index_p1 q72.sql timeout error

### PR DESCRIPTION
**Pull request by ZhangYu0123**
# Proposed changes

Fix inverted_index_p1 q72.sql timeout error
1、the runtime filter exeed wait time and lead to  100w * 1000w  data join
2、the error：
image

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ZhangYu0123** [comment]
run buildall

**Comment by ZhangYu0123** [comment]
run buildall

**Comment by ZhangYu0123** [comment]
run p0

**Comment by ZhangYu0123** [comment]
run buildall

**Comment by ZhangYu0123** [comment]
run buildall

File: regression-test/suites/inverted_index_p1/tpcds_sf1_index/ddl/call_center.sql [hunk 1]
```diff
@@ -1,8 +1,8 @@
 CREATE TABLE IF NOT EXISTS call_center (
     cc_call_center_sk bigint,
     cc_call_center_id char(16),
-    cc_rec_start_date date,
-    cc_rec_end_date date,
+    cc_rec_start_date datev2,
```
Comment (yiguolei) [inline regression-test/suites/inverted_index_p1/tpcds_sf1_index/ddl/call_center.sql:4]: do not need do this, we should submit a PR, to allow create a column with date type, but the underlying type is datev2

Comment (ZhangYu0123) [reply regression-test/suites/inverted_index_p1/tpcds_sf1_index/ddl/call_center.sql]: > do not need do this, we should submit a PR, to allow create a column with date type, but the underlying type is datev2

just commit something to run the test p1

**Comment by ZhangYu0123** [comment]
run buildall

**Comment by ZhangYu0123** [comment]
run buildall

**Comment by ZhangYu0123** [comment]
run p0

**Comment by ZhangYu0123** [comment]
run p1

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**